prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -186,7 +191,7 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 				fi_addr[i] = (uintptr_t) (void *) conn;
 				break;
 			case FI_AV_TABLE:
-				fi_addr[i] = av->count + i;
+				fi_addr[i] = av->used;
```

Comment:
This would be easier to read if you had something like:

fi_addr[i] = ++av->used;

versus having the used incremented above.  In the case of errors, used will be left too large.